### PR TITLE
Few new responses for talk_tags

### PR DESCRIPTION
**Pull request by RichardBlonski**
#### Summary

 ```SUMMARY: Interface "Added a few new responses to the talk tags."```

### CODE REVIEW

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -354,7 +358,8 @@
             "you see",
             "see, <name_g>",
             "alright",
-            "that clear"
+            "that clear",
+			"alright"
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: This is already an option, two lines up

File: data/json/npcs/talk_tags.json [hunk 2]
```diff
@@ -71,7 +71,8 @@
             "I <really> need to get some water.",
             "<ill_die> if I don't drink something.",
             "Water... Is there an oasis nearby?",
-            "Did you know that lack of water kills faster than lack of rest?"
+            "Did you know that lack of water kills faster than lack of rest?",
+			"I need water"
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: Needs a period at the end

File: data/json/npcs/talk_tags.json [hunk 3]
```diff
@@ -243,7 +246,8 @@
             "I'm a free agent, more money that way.",
             "I prefer to work uninhibited by that kind of connection.",
             "I haven't found one that's good enough for me.",
-            "I don't belong to a faction, <name_g>."
+            "I don't belong to a faction, <name_g>.",
+			"I'm a lonewolf right now."
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: Lone wolf is two words usually

File: data/json/npcs/talk_tags.json [hunk 4]
```diff
@@ -558,7 +564,8 @@
         "I'm not a contortionist!",
         "I can't fit in your tiny human vehicle.",
         "This sucks.  I'm too big.",
-        "I don't like being a mutant crammed into your tiny little vehicle."
+        "I don't like being a mutant crammed into your tiny little vehicle.",
+		"I don't feel comfortable in this vehicle."
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: I'm not sure this one really gets across the reason the NPC can't fit

File: data/json/npcs/talk_tags.json [hunk 5]
```diff
@@ -751,7 +759,8 @@
         "Quiet down over there!",
         "Did you hear someone talking?",
         "Who said that?",
-        "Who's there?"
+        "Who's there?",
+		"Shhh.. I've heard something"
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: Should read: "Shhh... I heard something." - three dots in an ellipsis, no contraction, and full stop at end.

File: data/json/npcs/talk_tags.json [hunk 6]
```diff
@@ -774,6 +783,7 @@
         "What's there?  I heard",
         "Did you hear that?  Sounded like",
         "What is making that sound?  I can hear the"
+		"Shh.. something is coming"
```
Comment (I-am-Erk) [inline data/json/npcs/talk_tags.json]: An ellipsis has three dots, and you need a period.

**Comment by mlangsdorf** [comment]
Have you abandoned this?  Please update it with i-am-erk's requested changes.

File: data/json/npcs/talk_tags.json [hunk 7]
```diff
@@ -486,7 +491,8 @@
             "<name_g>!  Wait up!",
             "Wait up, <okay>?",
             "Let's talk, <name_g>!",
-            "Look, <name_g><punc> let's talk!"
+            "Look, <name_g><punc> let's talk!",
+			"Hey, I want to talk"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json]: Period in the end.

**Comment by ZhilkinSerg** [comment]
I cannot resolve conflicts and/or update requested changes by myself as repo is set to `unknown`:

image

Would you please recreate this pull request?

**Pull request closed without merging**